### PR TITLE
EASYOPAC-1256 - Implementation of Readers Compass.

### PR DESCRIPTION
**Pull request by bozeak**
https://inlead.atlassian.net/browse/EASYOPAC-1256

### CODE REVIEW

File: ding_readers_compass.info [hunk 1]
```diff
@@ -6,8 +6,13 @@ version = 7.x-1.0
 dependencies[] = ctools
 dependencies[] = ding_adgangsplatformen
 dependencies[] = ding_provider
+dependencies[] = ding_ting_frontend
 dependencies[] = features
 dependencies[] = page_manager
 features[ctools][] = page_manager:pages_default:1
 features[features_api][] = api:2
-features[page_manager_pages][] = ding_readers_compass
```
Comment (guddo) [inline ding_readers_compass.info:13]: Why is this removed?

File: ding_readers_compass.pages_default.inc [hunk 2]
```diff
@@ -1,113 +1,114 @@
-<?php
```
Comment (guddo) [inline ding_readers_compass.pages_default.inc:1]: Can you just change the lines needed instead of writing the whole file?

File: ding_readers_compass.info (hunk 1 above)
Comment (bozeak) [reply ding_readers_compass.info:13]: Fixed.

File: ding_readers_compass.pages_default.inc (hunk 2 above)
Comment (bozeak) [reply ding_readers_compass.pages_default.inc:1]: Don't know why, but this is something I can't control in this case :/

File: ding_readers_compass.module [hunk 3]
```diff
@@ -16,12 +18,3 @@ function ding_readers_compass_ctools_plugin_directory($module, $plugin) {
     return "plugins/$plugin";
   }
 }
-
```
Comment (guddo) [inline ding_readers_compass.module:19]: Why is this removed?

File: ding_readers_compass.pages_default.inc (hunk 2 above)
Comment (guddo) [reply ding_readers_compass.pages_default.inc:1]: It's OK you can't control it - but we should not have diff like this

File: ding_readers_compass.module (hunk 3 above)
Comment (bozeak) [reply ding_readers_compass.module:19]: Because this is automatically exported by Features module into ```ding_readers_compass.features.inc``` file.

Comment (guddo) [reply ding_readers_compass.module:19]: I dont think we should manipulate this either
